ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -4542,17 +4542,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-inplace-f7
-  reporter_config:
```

Comment:
Those removal are not expected.

Reply:
hmm, No idea this,  just 'make update',  do you know the reason?

Reply:
Those `reporter_config` are added explicitly. 
`make update` will not remove it if it's already there. 
If it's removed accidentally, e.g, while resolving conflicts, `make update` can't add it back, too.

You may need to branch from the latest master and reapply your changes.

Reply:
Thanks. download the file and rebuild, now it's fixed